lms/models/region.py

```diff
@@ -18,25 +18,21 @@ class Region:
 class Regions:
     """A collection of all the regions as an enum like object."""
 
-    # As region objects are frozen, these can be directly compared like enums
-    US = Region(code="us", name="Worldwide (U.S.)", authority="lms.hypothes.is")
-    CA = Region(code="ca", name="Canada", authority="lms.ca.hypothes.is")
-    ALL = US, CA
-
-    _AUTHORITY_MAP = {region.authority: region for region in ALL}
-    _CODE_MAP = {region.code: region for region in ALL}
-
     @classmethod
-    def from_code(cls, code: str) -> Region:
+    def from_code(cls, authority: str, code: str) -> Region:
```

Comment:
Had to add an `authority` argument to `from_code()` because we're no longer using a hard-coded mapping of two possible `code`'s to two possible `authority`'s